lld/test/ELF/linkerscript/orphan-sections-init.s

```diff
@@ -0,0 +1,30 @@
+# REQUIRES: x86
+# RUN: rm -rf %t && mkdir -p %t
+# RUN: split-file %s %t && cd %t
+
+# RUN: llvm-mc -filetype=obj -triple=x86_64 foo.s -o foo.o
+
+# RUN: ld.lld -r  foo.o -T script.ld -o foo_mc.o
+
+# RUN: llvm-objcopy --rename-section .text=.com.text foo_mc.o foo_mc.o
+# RUN: llvm-objcopy --rename-section .rela.text=.rela.com.text foo_mc.o foo_mc.o
+
+# RUN: ld.lld -r foo_mc.o  -T script.ld -o foo_mc_after.o
+
+#--- foo.s
+  .text
+  .globl	foo
+  .p2align	4
+  .type	foo,@function
+foo:
+  mov $bar, %rax
+
+
```

Comment:
I recommend a CHECK of the llvm-readelf --sections output to make sure lld has created the link from the relocations correctly.

I've seen some scripts been run to find tests with no FileCheck output before so it is worth putting at least one in.